src/SiteSettings.php

```diff
@@ -262,6 +263,7 @@ public function setDevRedisSettings(): void {
     $this->settings['redis.connection']['interface'] = 'PhpRedis';
     $this->settings['redis.connection']['host'] = '127.0.0.1';
     $this->settings['cache']['default'] = 'cache.backend.redis';
+    $this->settings['cache_ttl_4xx'] = self::CACHE_404_TTL;
```

Comment:
This would only work in local development. How about production, do we want to add it there, too?

Reply:
Yes otherwise redis gets spammed with 404 unlimited caches and removes more important cache entries